Lib/_pyrepl/console.py

```diff
@@ -136,3 +139,45 @@ def wait(self) -> None:
 
     @abstractmethod
     def repaint(self) -> None: ...
+
+
+class InteractiveColoredConsole(code.InteractiveConsole):
```

Comment:
I moved this here so it can be imported even when `_pyrepl.readline` doesn't import (which is imported by `_pyrepl.simple_interact`).